tasks.py

```diff
@@ -201,114 +156,25 @@ def release(ctx, function=None, description=None):
     """
     Publish a new version of the function(s) and update the release alias to point to it
     """
-    if function is not None:
-        functions = [function]
-    else:
-        functions = FUNCTION_NAMES
-
+    functions = resolve_function_arg(function)
     for func in functions:
         new_version = __publish_version(ctx, func, description)
         __update_release_alias(ctx, func, new_version, description)
 
 
-@task
-def list_recordings(ctx, date=None):
-    """
-    Optional: --date='YYYY-MM-DD'
-    """
-    if date is None:
-        date = datetime_date.today()
-
-    meetings = __get_meetings(date)
-
-    recordings_found = 0
-
-    for meeting in meetings:
-        uuid = meeting['uuid']
-        series_id = meeting['id']
-
-        r = zoom_api_request(
-            "meetings/{}".format(series_id), ignore_failure=True,
-        )
-        if r.status_code == 404:
-            continue
-        r.raise_for_status()
-
-        r = zoom_api_request(
-            "meetings/{}/recordings".format(uuid), ignore_failure=True
-        )
-        if r.status_code == 404:
-            continue
-        r.raise_for_status()
-
-        # sometimes this zoom endpoint sends an empty list of recording files
-        if not len(r.json()['recording_files']):
-            continue
-
-        recordings_found += 1
-
-        local_tz = timezone(getenv('LOCAL_TIME_ZONE'))
-        utc = timezone('UTC')
-        utc_start_time = r.json()['recording_files'][0]['recording_start']
-        start_time = utc.localize(datetime.strptime(utc_start_time,
-                                  "%Y-%m-%dT%H:%M:%SZ")).astimezone(local_tz)
-
-        print("\n\tuuid: {}".format(uuid))
-        print("\tSeries id: {}".format(r.json()["id"]))
-        print("\tTopic: {}".format(r.json()["topic"]))
-        print("\tStart time: {}".format(start_time))
-        print("\tDuration: {} minutes".format(r.json()["duration"]))
-
-    if recordings_found == 0:
-        print("No recordings found on {}".format(date))
-    else:
-        print("Done!")
-
-
 @task
 def update_requirements(ctx):
     """
     Run a `pip-compile -U` on all requirements files
     """
     req_file = relpath(join(dirname(__file__), 'requirements.in'))
-    ctx.run("pip-compile -r -U {}".format(req_file))
-    for func in FUNCTION_NAMES:
+    ctx.run("pip install -U pip-tools && pip-compile -r -U {}".format(req_file))
```

Comment:
one time i got some errors because of a `pip-complile` vs `pip` compatibility thing. I think it's good to make sure they're both up-to-date.